### PR TITLE
Added translations and improved project structure

### PR DESCRIPTION
**Pull request by TungusSs**
Greetings, this is my first PR, please be patient if I did something wrong 😄.
I added Russian translation of README and Russian flag icon in the main README to indicate availability of translated version (README-ru.md). This is done to improve accessibility for Russian-speaking users. Also I made a `docs` folder, where all the README translations will be kept. If you want, you can make it for other documents (not only README) in the future.
Please forgive me for misprints or translation errors, if I missed something, please point out my mistakes, I will be grateful to you and I will correct them, also you can commit in my PR too.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Contributing translations:
```
Comment (TallTed) [inline CONTRIBUTING.md:1]:
```suggestion
# Translations of this page
```

File: README.md [hunk 2]
```diff
@@ -5,16 +5,19 @@
...
-    <a href="https://github.com/sponsors/AykutSarac" target="_blank"><img src="https://user-images.githubusercontent.com/47941171/206397875-a4e73f02-5d8f-4db0-902b-9a4bc2b22d90.png" "style=" height: 54px;" height="54" /></a>
+    <a href="https://github.com/sponsors/AykutSarac" target="_blank"><img src="https://user-images.githubusercontent.com/47941171/206397875-a4e73f02-5d8f-4db0-902b-9a4bc2b22d90.png" style=" height: 54px;" height="54"  alt="Sponsor us"/></a>
   </p>
 </div>
 
   <p align="center">
       <img width="800" src="./public/assets/jsoncrack-screenshot.webp" alt="preview 1" />
   </p>
 
+# Readme translations:
```
Comment (TallTed) [inline README.md:18]:
```suggestion
# Translations of this page
```

**Review by TallTed — APPROVED** [review]
The English portions look good to me. I can make no judgement of the Russian.

**Comment by AykutSarac** [comment]
Hi, thank you for the PR but for now we can keep the docs in English. We might bring localization to the project in the future though 🙂

**Pull request closed without merging**